draft-trammell-taps-post-sockets.md

```diff
@@ -950,8 +966,8 @@ limitations of 64kB and 16kB, respectively. Message sizes may also be limited by
 the available buffer at the receiver, since a Message must be fully assembled by
 the transport layer before it can be passed on to the application layer. Since
 not every transport protocol stack implements the signaling necessary to
-negotiate or expose message size limitations, these are currently configured out
-of band, and are probably best exposed through the policy context.
+negotiate or expose message size limitations, these are currently negotiated out
```

Comment:
s/negotiated/determined? (redundant use of negotiate)

Reply:
Reworded